docs/hyperopt.md

```diff
@@ -343,6 +343,10 @@ def populate_buy_trend(self, dataframe: DataFrame) -> DataFrame:
     return dataframe
 ```
 
+You can use the `--print-all` command line option if you would like to see all results in the hyperopt output, not only the best ones.
+
+When the `--color/--print-colorized` command line option is used, the results are colorized -- bad results (with zero trades or limited by the `--min-trades` option) are red, current bests -- in green, results with positive total profit are printed in bold.
```

Comment:
You will want to change this to `--no-color` too please

Reply:
Sure, I've forgotten about the docs, my bad...
Done.

Reply:
that's why we do reviews :+1: